openapi/v3/v3-connectors-config.yaml

```diff
@@ -54,9 +54,6 @@ components:
                     type: string
                 name:
                     type: string
-                pageSize:
-                    type: integer
-                    default: 25
```

Comment:
I suppose the "right" thing to do would be to mark it as deprecated.

Reply:
that's true, do we want to stick to the rule? There's no impact on console and fctl